hass_nabucasa/cloud_api.py

```diff
@@ -155,6 +156,58 @@ async def async_voice_connection_details(cloud: Cloud[_ClientT]) -> ClientRespon
     )
 
 
+@_check_token
+async def async_files_download_details(
+    cloud: Cloud[_ClientT],
+    *,
+    storage_type: str,
+    filename: str,
+) -> dict[str, Any]:
```

Comment:
Would be awesome if we could make this a TypedDict

Reply:
https://github.com/NabuCasa/hass-nabucasa/pull/640